eng/versioning/update_versions.py

```diff
@@ -275,6 +296,7 @@ def main():
     parser.add_argument('--auto-version-increment', '--avi', action='store_true', help='If this script is being run after an auto version increment, add changelog entry for new version')
     # Comma separated list artifacts, has to be split into an array. If we're not skipping README updates, only update MD files for entries for the list of libraries passed in
     parser.add_argument('--library-list', '--ll', nargs='?', help='(Optional) Comma seperated list of groupId:artifactId. If updating MD files, only update entries in this list.')
+    parser.add_argument('--version_override', '--vo', nargs='?', help='(Optional) identifier of version update configuratation matching entry in alternative_external_dependency_versions.json')
```

Comment:
Should indicate that is is expected to be an array (a coma-delimited array?)

Reply:
it's not  an array, but I tried to clarify the description